repository/src/main/java/org/apache/atlas/tasks/TaskManagement.java

```diff
@@ -52,6 +52,11 @@ public class TaskManagement implements Service, ActiveStateChangeHandler {
 
     private Thread watcherThread = null;
 
+    public enum DeleteType {
```

Comment:
Now, I realized that there is already a DeleteType enum in project. Should I remove this and use that instead?